src/queries/project.js

```diff
@@ -1,38 +1,34 @@
-// Project Queries
-// Select all Projects of a user
 export const FETCH_All_PROJECTS = (`
-    select * 
-    from users u 
-    left join user_project up on up.user_id = u.id 
-    left join projects p on up.project_id = p.id 
-    where u.id = ? ;
+    SELECT * 
+    FROM users u 
+    LEFT JOIN user_project up on up.user_id = u.id 
+    LEFT JOIN projects p on up.project_id = p.id 
+    WHERE u.id = ? ;
     `);
 
-// Select a project from a user when project id id given
 export const FETCH_A_PROJECT = (`
-    select * 
-    from users u 
-    left join user_project up on up.user_id = u.id 
-    left join projects p on up.project_id = p.id 
-    where u.id = ? 
+    SELECT * 
+    FROM users u 
+    LEFT JOIN user_project up on up.user_id = u.id 
+    LEFT JOIN projects p on up.project_id = p.id 
+    WHERE u.id = ? 
     AND p.id = ?;
     `);
 
-// Delete project entry from projects table
 export const DELETE_A_PROJECT_PROJECTS = (`
-    Delete
-    from projects
-    where id=?
+    DELETE
+    FROM projects
+    WHERE id=?
     `);
 
 export const DELETE_A_PROJECT_USERPROJECT = (`
-    Delete
-    from user_project
-    where project_id=?
+    DELETE
```

Comment:
Use raw queries ONLY for complex `SELECT` queries that affects readability if done with query builder. Not for `UPDATE`, `DELETE`, `INSERT` and trivial queries like `SELECT * FROM table`.